cs-jclouds/src/main/java/io/cloudslang/content/jclouds/actions/instances/ModifyInstanceAttributeAction.java

```diff
@@ -0,0 +1,217 @@
+package io.cloudslang.content.jclouds.actions.instances;
```

Comment:
I agree with you... but not in this PR... should follow-up another one only for updating names/packages so on... :smile: